config/webhook/manifests.yaml

```diff
@@ -1,4 +1,32 @@
 
+---
```

Comment:
whats specific to windows here? again just thinking out loud as i go through it

Reply:
just the auto-generation at work